bgpd/bgp_fsm.c

```diff
@@ -2163,6 +2163,9 @@ bgp_establish(struct peer_connection *connection)
 	peer->established++;
 	bgp_fsm_change_status(connection, Established);
 
+	if (peer->last_reset == PEER_DOWN_WAITING_OPEN)
```

Comment:
I really don't get this change.  Can you help me understand why displaying this is not good?  And if we don't want to use PEER_DOWN_WAITING_OPEN why keep it?

Reply:
When the peer is created, "last reset" is displayed as "Waiting for OPEN" even when the connection is established which is something a "confusing part" (because it wasn't reset actually).